doc/api/os.md

```diff
@@ -2,25 +2,36 @@
 
     Stability: 2 - Stable
 
-Provides a few basic operating-system related utility functions.
+The `os` module provides a number of operating system-related utility methods.
+It can be accessed using:
 
-Use `require('os')` to access this module.
+```js
+const os = require('os');
+```
 
 ## os.EOL
 <!-- YAML
 added: v0.7.8
 -->
 
-A constant defining the appropriate End-of-line marker for the operating
-system.
+A string constant defining the operating-system specific end-of-line marker:
+
+* `\n` on POSIX
+* `\r\n` on Windows
 
 ## os.arch()
 <!-- YAML
 added: v0.5.0
 -->
 
-Returns the operating system CPU architecture. Possible values are `'x64'`,
-`'arm'` and `'ia32'`. Returns the value of [`process.arch`][].
+The `os.arch()` method returns a string identifying the operating system CPU
```

Comment:
"operating system CPU architecture" sounds a bit odd/misleading. It's really the architecture that the node binary was compiled for (since the value is baked in at compile time), which *could* be different than the current CPU architecture. For example, a node binary compiled for x86 could still be run on an x64 CPU, however `process.arch`/`os.arch()` would still return `x86`.